### DISCUSSION TITLE
Homing cycle problem.

### DISCUSSION THREAD

**Discussion by Marmadoc** [winder/Universal-G-Code-Sender#2492] [Help]
I try to use setup wizard and I'm stuck in Homing cycle.
![1](https://github.com/winder/Universal-G-Code-Sender/assets/69551886/07955531-31ec-46d0-bcac-b0c974cf65b7)

When I press "Try Homing" both steppers are powered on, holds the torque, but they didn't move. After certain time Alarm is triggered.
I use two axis homing:
#define HOMING_CYCLE_0 ((1<<X_AXIS)|(1<<Y_AXIS))
and my GRBL settings is
![Untitled2](https://github.com/winder/Universal-G-Code-Sender/assets/69551886/73ba2569-cba4-4adb-bac8-751cae19e778)
Can somebody help me.
Thank You

**Comment by breiler** [comment]
Are you able to unlock the machine and jog it manually?

**Reply by Marmadoc** [reply]
Yes - jog manually is perfect. Limit switches works OK.

**Reply by breiler** [reply]
Your homing seek rate is set to 500mm/min, can you jog both X/Y axes diagonally in 500mm/min?

**Reply by Marmadoc** [reply]
Yes - no problem
![Untitled3](https://github.com/winder/Universal-G-Code-Sender/assets/69551886/33b7e424-5fb7-4d61-adcf-ae459d2ad931)

**Reply by breiler** [reply]
Hmm, sorry no idea what is going on.

**Reply by Marmadoc** [reply]
I like to let you know that I switch back to GRBL(i) (not h hex file) - loaded via Arduino IDE - and now homing cycle is working (Home button) works perfectly.
